main.go

```diff
@@ -106,6 +106,68 @@ func signTxCmd() *ffcli.Command {
 	}
 }
 
+func batchSignTxCmd() *ffcli.Command {
+	fs := flag.NewFlagSet("sign-batch", flag.ContinueOnError)
+	keyringDir := fs.String("keyring-dir", "", "Keyring directory (mandatory with -keyring-backend=file)")
+	keyringBackend := fs.String("keyring-backend", "", "Keyring backend, which can be one of 'keychain' (macos), 'pass', 'kwallet' (linux), or 'file'")
+	signer := fs.String("from", "", "Signer key name")
+	chainID := fs.String("chain-id", "", "Chain identifier")
+	account := fs.String("account", "", "Account number")
+	sequence := fs.String("sequence", "", "Sequence number")
+	sigOnly := fs.Bool("signature-only", false, "Outputs only the signature data")
```

Comment:
Let's have a common function to set sign flags between `sign-tx` and `sign-batch`.